lib/elastic_record/index/documents.rb

```diff
@@ -155,6 +155,16 @@ def search(elastic_query, options = {})
         get url, elastic_query.update('_source' => load_from_source)
       end
 
+      def multi_search(elastic_queries)
+        url = "_msearch"
```

Comment:
No reason for the `url` variable. This is made in `search` because of additional logic that adds options.